appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -100,7 +100,7 @@ Blockly.ReplMgr.isConnected = function() {
  * Build YAIL for sending to the companion.
  * @param {Blockly.WorkspaceSvg} workspace
  */
-Blockly.ReplMgr.buildYail = function(workspace) {
+Blockly.ReplMgr.buildYail = function(workspace, optForce) {
```

Comment:
I would prefer `opt_force` rather than `optForce`. Add the description of the parameter to the JsDoc as well. The type should be `{boolean=}`.